scripts/Dockerfile

```
@@ -17,7 +17,7 @@ ARG BUILD_CONCURRENCY="0"
 
 #Install dependencies, eliminate annoying warnings
 RUN sed -i -E -e 's/include <sys\/poll.h>/include <poll.h>/' /usr/include/boost/asio/detail/socket_types.hpp
-RUN cmake -DCMAKE_BUILD_TYPE=Release -DTESTS=0 -DSOLC_LINK_STATIC=1
+RUN CC=$(which clang) CXX=$(which clang++) cmake -DCMAKE_BUILD_TYPE=Release -DTESTS=0 -DSOLC_LINK_STATIC=1
```

I don't think you need an absolute path here. I've always been doing it like this:

```suggestion
RUN cmake -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_BUILD_TYPE=Release -DTESTS=0 -DSOLC_LINK_STATIC=1
```

---

Absolutely, any of the permutations spanned by `absolute/relative path` x `environment variable/cmake option` works :) Fixed!